cdp_scrapers/legistar_utils.py

```diff
@@ -507,12 +508,17 @@ def get_time_zone(self) -> str:
         Returns
         -------
         time zone name : str
-            i.e. "US/Pacific" | "America/Los_Angeles" | ...
+            i.e. "America/Los_Angeles" | "America/New_York" ...
 
         See Also
         --------
         SeattleScraper.get_time_zone()
 
+        Notes
+        -----
+        List of Timezones: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+        Please only use "Canonical" timezones.
+
         """
         log.error(
             "time zone name e.g. US/Pacific " "required for proper event timestamping"
```

Comment:
Guess might as well also change the error message to say "America/Los_Angeles"

Reply:
Done! Nice catch.